metadata/metadata_service/proxy/neo4j_proxy.py

```diff
@@ -1709,19 +1709,17 @@ def _exec_feature_query(self, *, feature_key: str) -> Dict:
 
         feature_query = textwrap.dedent("""\
         MATCH (feat:Feature {key: $feature_key})
-        OPTIONAL MATCH (db:Database)-[:FEATURE]->(feat)
+        OPTIONAL MATCH (db:Database)-[:AVAILABLE_FEATURE]->(feat)
+        OPTIONAL MATCH (fg:Feature_Group)-[:GROUPS]->(feat)
         OPTIONAL MATCH (feat)-[:LAST_UPDATED_AT]->(t:Timestamp)
         OPTIONAL MATCH (feat)-[:OWNER]->(owner:User)
         OPTIONAL MATCH (feat)-[:TAGGED_BY]->(tag:Tag)
         OPTIONAL MATCH (feat)-[:HAS_BADGE]->(badge:Badge)
-        OPTIONAL MATCH (feat)-[:COLUMN]->(col:Column)-[:HAS_BADGE]->(col_badge:Badge)
-        OPTIONAL MATCH (col)-[:DESCRIPTION]->(col_desc:Description)
         OPTIONAL MATCH (feat)-[:DESCRIPTION]->(desc:Description)
         OPTIONAL MATCH (feat)-[:DESCRIPTION]->(prog_descriptions:Programmatic_Description)
-        OPTIONAL MATCH (wmk:Watermark)-[:BELONG_TO_TABLE]->(feat)
+        OPTIONAL MATCH (wmk:Feature_Watermark)-[:BELONG_TO_FEATURE]->(feat)
         RETURN feat, collect(distinct wmk) as wmk_records,
```

Comment:
nit, can `collect(distinct wmk)` be below next to the other `collect(distinct ...)`